src/interfaces/packagingInterfacesAndType.ts

```diff
@@ -283,6 +287,7 @@ export type PackageVersionCreateOptions = {
   definitionfile: string;
   installationkey: string;
   installationkeybypass: boolean;
+  language: string;
```

Comment:
The location field needs to be added to the appropriate sobject found in packagingSObjects.ts

Reply:
Thank you @peternhale . Not sure how I missed that in my local repo.

Reply:
Done